components/scream/data/scream_input.yaml

```diff
@@ -38,7 +38,7 @@ Atmosphere Driver:
 
   Initial Conditions:
     Physics GLL:
-      Filename: ./data/init_ne4np4.nc
+      Filename: "<${COMPSET} : .*SCREAM%AQUA.* => ${DIN_LOC_ROOT}/atm/scream/init/scream-aquaplanet_init_ne4np4_L72_20211202.nc : ./data/init_ne4np4.nc>"
```

Comment:
Okay cool. Now we pass all v0 tests but still have a DIFF in the original SCREAMV1 test. Is it possible this logical is not working correctly?  Just trying to track down the last unexpected DIFF in the AT results.

Reply:
It appears to be working correctly, I just ran this test manually and it gets the proper (old) initial condition. But, then I ran this test on current master, and the v1 test is DIFF'ing on current SCREAM master for me. @bartgol was also tracking down another unexpected DIFF from the v1 tests with #1356 and suspected something funny with baselines.